typescript/src/types/Activity.ts

```diff
@@ -4,19 +4,20 @@ enum Activity {
   DevDebug = "DevDebug",
   DevReview = "DevReview",
   DevVc = "DevVc",
-  Email = "Email",
   Planning = "Planning",
   ReadWriteDocument = "ReadWriteDocument",
+  Design = "design",
```

Comment:
Is there a reason why this is lowercase?

Reply:
thanks for the hint, fixed